recipe/meta.yaml

```diff
@@ -29,9 +26,20 @@ requirements:
     - {{ compiler('cxx') }}
     - bazel 
   host:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - bazel 
     - python >=3.8
     - pip
-    - bazel
+    - tensorflow-base >=2.13.0, <2.14.0
+    - jax >=0.4.14, <0.5.0
+    - numpy >=1.13.3
+    - absl-py
+    - six >=1.10.0
+    - decorator
+    - cloudpickle >=1.3
+    - gast >=0.3.2
+    - dm-tree
```

Comment:
There is any reason you added all the run dependencies also in the host? If not necessary it would be great not to have them here, otherwise all the constraints need to be updated manually also here.

Reply:
I initially thought they were necessary for the bazel build as it is running some Python scripts underneath, sorry. I'll remove them